src/Cake.Common/Net/HttpAliases.cs

```diff
@@ -197,7 +197,7 @@ public static void DownloadFile(this ICakeContext context, Uri address, FilePath
             // We track the last posted value since the event seems to fire many times for the same value.
             var percentComplete = 0;
 
-            using (var http = new HttpClient())
+            using (var http = new HttpClient(new HttpClientHandler { UseDefaultCredentials = settings.UseDefaultCredentials }))
```

Comment:
I would say that this makes no sense. The current implementation is using Basic Authentication. My pull request is about supporting NTLM. Maybe we should abstract both into a separate SettingsObject i.e. BasicAuthenticationSettings and NTLMAuthenticationSettings.